### ISSUE TITLE
Problems with styles

### ISSUE THREAD

**Issue by markwilliams1234** [TurboPack/RibbonFramework#86]
![untitled](https://user-images.githubusercontent.com/35732517/37915867-642135c2-3112-11e8-8f50-bc28db9e379f.png)
I have noticed two different problems with loading styles (either as default or whilst running an app). In the case of the Textpad 06 sample app, if you load a style by default or choose a new style whilst running you lose the quick access toolbar and you also get a space above the ribbon (see attached png).

In an app of my own, if I set a style by default, I have found I get different problems with different forms:

The Main form - works fine (including the quick access toolbar is visible), but the form frame does not adopt the chosen style.
Subsidiary forms present the same issues as the textpad form.

I can't see any significant difference between the main form in my app and the Textpad app.

**Comment by markwilliams1234** [comment]
I have discovered the difference between the two apps. A styled form will show the quick access toolbar if you turn off seBorder in its style elements. However, then the frame and caption do not get styled. If you don't turn them off, you lose the quick access toolbar and get a blank space above the ribbon.

**Comment by joachimmarder** [comment]
I don't think that VCL styles and the Windows Ribbon Framework are compatible. The Windows Ribbon framework is basically an embedded ActiveX control that does some additional magic in the window title, I don't think it can be themed, it will be themed according to the Windows version.
I also tried if https://github.com/RRUZ/vcl-styles-utils helps, but it did not change anything.  Maybe explicit support needs to be added for the Windows Ribbon Framework window classes.

**Comment by markwilliams1234** [comment]
For time being then it would seem sensible to disable styles for form frame. Shame. Thanks
